lib/bloc/lnurl/lnurl_bloc.dart

```diff
@@ -64,8 +64,7 @@ class LNUrlBloc with AsyncActionsHandler {
         })
       ])
           .where((l) => l != null)
-          .map((l) => l.toLowerCase())
-          .where((l) => isLNURL(l) || isLightningAddress(l))
+          .where((l) => isLNURL(l.toLowerCase()) || isLightningAddress(l.toLowerCase()))
```

Comment:
Can be reduced to:
`.where((l) => l != null && (isLNURL(l) || isLightningAddress(l)))`